corehq/apps/domain/views/releases.py

```diff
@@ -181,3 +185,30 @@ def update_release_restriction(request, domain, restriction_id, active):
                                if release.deactivated_on else None),
         }
     return HttpResponse(json.dumps(response_content), content_type='application/json')
+
+
+def toggle_release_restriction_by_app_profile(request, domain, restriction_id):
+    if not toggles.RELEASE_BUILDS_PER_PROFILE.enabled_for_request(request):
+        return HttpResponseForbidden()
+    release = LatestEnabledBuildProfiles.objects.get(id=restriction_id)
+    if not release:
+        return HttpResponseBadRequest()
+    if request.POST.get('active') == 'false':
+        return _update_release_restriction_by_app_profile(request, restriction_id, active=False)
+    elif request.POST.get('active') == 'true':
+        return _update_release_restriction_by_app_profile(request, restriction_id, active=True)
+
+
+def _update_release_restriction_by_app_profile(release, restriction_id, active):
+    try:
+        release.activate() if active else release.deactivate()
+    except ValidationError as e:
+        response_content = {
+            'message': ','.join(e.messages)
+        }
+    else:
+        response_content = {
+            'id': restriction_id,
+            'success': True,
+        }
+    return HttpResponse(json.dumps(response_content), content_type='application/json')
```

Comment:
nit: you can also use `django.http.JsonResponse`

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/7c0eadac8a4bc17703cf2d6618b038eca4a99e6d

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/fa6e66c9551c9ba1147f5260b53ab7ab720a09fb